front/app/component-library/components/Checkbox/index.tsx

```diff
@@ -27,6 +27,7 @@ const StyledCheckbox = styled.div<{
   checkedColor?: Color;
   size: string;
   disabled: boolean;
+  usePrimaryBorder?: boolean;
```

Comment:
We don't want to use the primary colour as the border for back office (for example). So I've added a prop to the Checkbox and Radio buttons specifically for when we want to use the primary colour for the border.

Reply:
Could you add this in the code itself (above the prop definition), and include reasoning (that it was added for a11y, when it should be used, ...)? :) Also thinking: if there's no strong reason for not using it in the admin, wouldn't it be easier if we always used it? Now we need to think about it each time we use the checkbox in the front-office.

Reply:
Or alternatively: turn it on by default, then pass usePrimaryBorder={false} in admin/when we don't want it. Otherwise this problem might propagate again throughout the codebase.

Reply:
> that it was added for a11y, when it should be used

@brentguf This was purely a cosmetic change actually :) The grey was fine before, but Rob wondered if we could have the native survey multiselect questions be more branded if we were already making changes to the colours anyways.

> Also thinking: if there's no strong reason for not using it in the admin, wouldn't it be easier if we always used it? Now we need to think about it each time we use the checkbox in the front-office.

Personally I feel like the default grey we have today looks best in most cases :thinking: In the back office we don't use the tenant primary colour at all I believe (only the cl-admin colours), so I think it's best not to start using it there?

Reply:
Aah, I thought it was about a11y (didn't read the whole conversation). Fine then :)